core/components/css-utilities/Border/Border.story.tsx

```diff
@@ -129,50 +218,323 @@ export const border = () => {
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border-bottom bg-secondary-lightest p-8 d-inline-block mr-6">bottom</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border-left bg-secondary-lightest p-8 d-inline-block">left</Text>'
+              }
+            </code>
+          </pre>
+        </div>
+        <Text appearance="subtle" className="mb-2 font-size-xs">
+          These classes allow you to add borders to specific sides of an element.
+        </Text>
+      </div>
+
+      {/* Border removal examples */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          Border Removal
+        </Heading>
+        <div className="mb-4">
+          <Text appearance="default" className="border border-0 bg-secondary-lightest p-8 d-inline-block mr-6">
+            border-0
+          </Text>
+          <Text appearance="default" className="border border-top-0 bg-secondary-lightest p-8 d-inline-block mr-6">
+            top-0
+          </Text>
+          <Text appearance="default" className="border border-right-0 bg-secondary-lightest p-8 d-inline-block mr-6">
+            right-0
+          </Text>
+          <Text appearance="default" className="border border-bottom-0 bg-secondary-lightest p-8 d-inline-block mr-6">
+            bottom-0
+          </Text>
+          <Text appearance="default" className="border border-left-0 bg-secondary-lightest p-8 d-inline-block">
+            left-0
+          </Text>
+        </div>
+        <div className="mb-5 bg-secondary-lightest p-3 rounded-10 font-size-xs">
+          <pre className="m-0">
+            <code className="d-block">
+              {
+                '<Text appearance="default" className="border border-0 bg-secondary-lightest p-8 d-inline-block mr-6">border-0</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-top-0 bg-secondary-lightest p-8 d-inline-block mr-6">top-0</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-right-0 bg-secondary-lightest p-8 d-inline-block mr-6">right-0</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-bottom-0 bg-secondary-lightest p-8 d-inline-block mr-6">bottom-0</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="border border-left-0 bg-secondary-lightest p-8 d-inline-block">left-0</Text>'
```

Comment:
Instead of using multiple <code> elements, it's better to place all the text within a single <code> block.